src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -176,7 +176,14 @@
 K8s_Cluster_Info = "k8s_cluster_info.txt"
 Outbound_Network_Connectivity_Check = "outbound_network_connectivity_check.txt"
 Events_of_Incomplete_Diagnoser_Job = "diagnoser_failure_events.txt"
-
+# Connect Precheck Diagnoser constants
+Pre_Onboarding_Inspector_Job_Registry_Path = "connectprecheck.azurecr.io/helm/pre-onboarding-inspector:0.1.0"
+Pre_Onboarding_Inspector_Check_Failed_Fault_Type = "Error occured while running the pre onboarding inspector"
```

Comment:
this is when one of the checks fail or when we were not able to run the check due to some issue?

Reply:
correct , this is used in the exception handling in the check_preonboarding_inspector_container function in the precheckutils

Reply:
then we need to rename the variable name. Check_failed is indicating that the DNS or network check failed. It might be confusing

Reply:
We have it the same way in the troubeshootutils code , i was trying to keep it consistent with that
" Diagnoser_Container_Check_Failed_Fault_Type = "Error occured while performing the diagnoser container checks" "

Reply:
Hmm okay, but ideally that also should've been corrected. We can correct it for pre-checks for now